tensorboard/webapp/metrics/views/main_view/main_view_component.scss

```
@@ -160,3 +161,18 @@ limitations under the License.
     margin-right: 4px;
   }
 }
+
+.slide-out-menu {
+  position: absolute;
+  height: calc(100% - 50px);
+  top: 50px;
+  left: calc(100vw - 250px);
+  width: 200px;
+  background-color: white;
+  border: 1px solid mat.get-color-from-palette(mat.$gray-palette, 300);
+  transition: all 0.75s ease;
+}
+
+.slide-out-menu-expanded {
+  left: calc(100vw - 450px);
+}
```

It's hard to capture but this existing leads to weird behavior when resizing the page.
Two things:
1) This doesn't need to be visible when not expanded.
2) There are a lot of calls to `calc` here ad I don't think you actually need any of them. You will also need to give `.split-content` `position: relative;` on line 75.
```suggestion
.slide-out-menu {
  @include tb-theme-background-prop(background-color, selected-button);
  position: absolute;
  height: 100%;
  right: 250px;
  width: 0;
  transition: all 0.75s ease;
  visibility: hidden;
}

.slide-out-menu-expanded {
  border-left: 1px solid mat.get-color-from-palette(mat.$gray-palette, 300);
  width: 200px;
  visibility: visible;
}
```

---

I changed this to not need calc and fixed the delayed moving issue you discussed.

Also reordered properties in alphabetic order.